parts/kuberneteswindowssetup.ps1

```diff
@@ -178,10 +178,14 @@ Write-KubernetesStartFiles($podCIDR)
 c:\k\kubelet.exe --hostname-override=`$global:AzureHostname --pod-infra-container-image=kubletwin/pause --resolv-conf="" --allow-privileged=true --enable-debugging-handlers --api-servers=https://`${global:MasterIP}:443 --cluster-dns=`$global:KubeDnsServiceIp --cluster-domain=cluster.local  --kubeconfig=c:\k\config --hairpin-mode=promiscuous-bridge --v=2 --azure-container-registry-config=c:\k\azure.json --runtime-request-timeout=10m
 "@
 
+    if ($global:KubeBinariesVersion -ge "1.6.0" -and $global:KubeBinariesVersion -le "1.7.0")
+    {
+            $KubeletArgList += "--enable-cri=false"
+            $KubeletCommandLine += " --enable-cri=false"
+    }
     if ($global:KubeBinariesVersion -ne "1.5.3" -and $global:KubeBinariesVersion -ne "1.5.7")
     {
-        $KubeletArgList += "--enable-cri=false"
-        $KubeletCommandLine += " --enable-cri=false --image-pull-progress-deadline=20m --cgroups-per-qos=false --enforce-node-allocatable=`"`""
+        $KubeletCommandLine += " --image-pull-progress-deadline=20m --cgroups-per-qos=false --enforce-node-allocatable=`"`""

```

Comment:
@seanknox yes, 1.5.* doesn't have --image-pull-progress-deadline option. The timeout is hardcoded. We have that option from 1.6.0 and above.